ui/src/App.js

```diff
@@ -31,6 +31,7 @@ class App extends Component {
               });
           }
       }.bind(this);
+      // Map of facetName:[facetValues] pairs
```

Comment:
"pairs" is confusing -- where is the pair?

// Map from facet name to a list of facet values. 

Same on line 113

Reply:
Done